test/e2e-tests.sh

```diff
@@ -1,5 +1,16 @@
 #!/usr/bin/env bash
 
+if [ -n "$OPENSHIFT_CI" ]; then
```

Comment:
This will have to stay here for this to be mergable as the build-image will only update after a merge to main. We can then drop it in a subsequent PR.